blueprint/aws-sam-email-classifier/template.yaml

```diff
@@ -0,0 +1,63 @@
+AWSTemplateFormatVersion: '2010-09-09'
+Transform: AWS::Serverless-2016-10-31
+Description: >
+  sam-app
+
+  Sample SAM Template for email-classifier
+
+Globals:
+  Function:
+    Timeout: 3
+    MemorySize: 128
+
+Resources:
+  ApiGatewayEndpoint:
+    Type: AWS::Serverless::Api
+    Properties:
+      StageName: Prod
+      Auth:
+        ApiKeyRequired: true
+        UsagePlan:
+          CreateUsagePlan: PER_API
+          UsagePlanName: GatewayAuthorization
+
+  EmailClassifierFunction:
+    Type: AWS::Serverless::Function # More info about Function Resource: https://github.com/awslabs/serverless-application-model/blob/master/versions/2016-10-31.md#awsserverlessfunction
+    Properties:
+      CodeUri: emailclassifier/
+      Handler: app.lambda_handler
+      Runtime: python3.9
```

Comment:
can we update the runtime to python3.12 if possible?

Reply:
Python 3.9 is required when using AWS SAM. I have tried to use later versions, but it seems to be incompatible.

Reply:
Agree with Marlowe